python/artm/dictionary.py

```diff
@@ -154,7 +157,7 @@ def gather(self, data_path, cooc_file_path=None, vocab_file_path=None, symmetric
                                        symmetric_cooc_values=symmetric_cooc_values)
 
     def filter(self, class_id=None, min_df=None, max_df=None, min_df_rate=None, max_df_rate=None,
-               min_tf=None, max_tf=None, max_dictionary_size=None):
+               min_tf=None, max_tf=None, max_dictionary_size=None, recalculate_value=False):
```

Comment:
Note: recalculate_value == True is useful in dictionary for perplexity calculation, False - for dictionary with custom 'value' values for regularization